tools/analyzer_plugin/lib/src/diagnostic/boilerplate_validator.dart

```diff
@@ -253,16 +253,6 @@ extension on SourceSpan {
   SourceRange asRangeOrEmpty() => this == null ? SourceRange.EMPTY : asRange();
 }
 
-extension<E> on Iterable<E> {
-  void forEachIndexed(void Function(E, int) callback) {
```

Comment:
The new analyzer version made improvements to the analysis of extensions and caught this was unused